resources/istio-configuration/README.md

```diff
@@ -0,0 +1,42 @@
+# Istio Configuration
+
+## Overview
+
+[Istio](https://istio.io/) is an open-source service mesh providing a uniform way to integrate microservices, manage traffic flow across microservices, enforce policies, and aggregate telemetry data.
+
+The documentation here is for developers only, please follow the installation instructions from [istio.io](https://istio.io/docs/setup/install/istioctl/) for all other use cases.
+
+The Istio Configuration Helm chart consists of:
+
+- `istio-operator.yaml` file implementing Kyma-specific changes and configuration options,
+- Mutual TLS (mTLS) configuration enabling mTLS cluster-wide in a STRICT mode,
+- Istio monitoring configuration details.
+
+This chart can install the following Istio components:
```

Comment:
```suggestion
This chart can install the following Istio components:
```

Can it (optional/in some cases/just some of them) or does it (always?) What's the default?

Reply:
As line 21 says: "To enable or disable each component, change the corresponding `enabled` flag." I will rephrase it a little bit.